MApp.js

```diff
@@ -0,0 +1,110 @@
+document.addEventListener('DOMContentLoaded', () => {
+   fetch('https://ghibliapi.herokuapp.com/films') //we want the data here before user interacton
+   .then(response => response.json())
+
+   .then(dataObj => {
+      fetchMovie(dataObj)
+
+      // fetchMovie(response)
+   })
+
+   // .catch(err =>{
+   //    console.log("Nah Dawg", err)
+   // })
+
+   eventListeners()
+})
+
+var iDontBelong = [null] //API response 
+ //common endpoint
+function fetchMovie(film){  //code initially existed at the end
+   let button3 = document.querySelector('#sumnToWatch')
+   for(let i of film){
+   let filmData = document.createElement('option')
+      filmData.innerText = i.title
+
+      button3.appendChild(filmData)
+
+      iDontBelong.push(i.id)
+      }
+   }
+
+   //No idea on how to add default clear empty option
+const eventListeners = () => {
```

Comment:
This function is doing way too many things inside. It is important to keep your code more readable by creating smaller functions that only does one thing depending on argument input.